pkg/integrations/v2/app_agent_receiver/payload.go

```diff
@@ -207,11 +207,17 @@ func (l Log) KeyVal() *KeyVal {
 	return kv
 }
 
+// ExceptionContext is a string to string map structure that
+// represents the context of a log message
+type MeasurementContext map[string]string
+
 // Measurement holds the data for user provided measurements
 type Measurement struct {
 	Values    map[string]float64 `json:"values,omitempty"`
 	Timestamp time.Time          `json:"timestamp,omitempty"`
 	Trace     TraceContext       `json:"trace,omitempty"`
+	Context   MeasurementContext `json:"context,omitempty"`
+	Type      string             `json:"type,omitempty"`
```

Comment:
`Type` doesn't appear to be included in the `KeyVal` object. Do we want to add this to `func (m Measurement) KeyVal() *KeyVal` ? 

```go
KeyValAdd(kv, "type", m.Type)
```